test-util/src/main/scala/io/buoyant/test/Json.scala

```diff
@@ -0,0 +1,79 @@
+package io.buoyant.test
+
+import com.fasterxml.jackson.core.util.{DefaultIndenter, DefaultPrettyPrinter}
+import com.fasterxml.jackson.databind.SerializationFeature.ORDER_MAP_ENTRIES_BY_KEYS
+import com.fasterxml.jackson.databind.node.TextNode
+import com.fasterxml.jackson.databind.{JsonNode, ObjectMapper}
+import com.fasterxml.jackson.module.scala.experimental.ScalaObjectMapper
+import org.scalatest.exceptions.TestFailedException
+
+import scala.util.control.NonFatal
+
+trait Json { _: Logging =>
+
+  private val mapper = new ObjectMapper with ScalaObjectMapper
+  private lazy val prettyMapper = {
+    mapper.writer(new DefaultPrettyPrinter {
+      _arrayIndenter = DefaultIndenter.SYSTEM_LINEFEED_INSTANCE
+    })
+  }
+  private lazy val sortingMapper = (new ObjectMapper with ScalaObjectMapper).configure(ORDER_MAP_ENTRIES_BY_KEYS, true)
+
+  def assertJsonEquals(actualJson: String, expectedJson: String, verbose: Boolean = true): Unit = {
+    val actualJsonNode = tryJsonNodeParse(actualJson)
+    val expectedJsonNode = tryJsonNodeParse(expectedJson)
+    if (actualJsonNode != expectedJsonNode) {
+      println("JSON DIFF FAILED!")
```

Comment:
Since this is intended to be mixed into tests, we should use a scalatest primitive such as `assert` for output instead of println